lib/fs-driver.js

```diff
@@ -30,7 +30,9 @@ class FsDriver {
     if (!isString(str)) {
       str = str.toString()
     }
-    return str.replace(/(['/\\])/g, '\\$1')
+    return str.replace(/'/g, '\'"\'"\'')
```

Comment:
Unfortunately [yes](http://stackoverflow.com/questions/1250079/how-to-escape-single-quotes-within-single-quoted-strings)